src/esp/assets/ResourceManager.cpp

```diff
@@ -1237,7 +1251,10 @@ bool ResourceManager::loadRenderAssetGeneral(const AssetInfo& info) {
   assimpmetadata->configuration().setValue("ImportColladaIgnoreUpDirection",
                                            "true");
 #endif
-  {
+
+  if (!requiresTextures_) {
```

Comment:
Would it just be cleaner to remove the else and rephrase the comment here?

Reply:
Do you mean to move everything into `if(requiresTextures_)`?

Reply:
I think you understand @Skylion007  correctly. 
Would you change it?